tools/fix_020.py

```diff
@@ -0,0 +1,59 @@
+from datasets import load_dataset, Dataset, DatasetDict
+from huggingface_hub import HfApi
+
+import json
+import copy
+
+BIGCODEBENCH_HF = "bigcode/bigcodebench"
+BIGCODEBENCH_HARD_HF = "bigcode/bigcodebench-hard"
+BIGCODEBENCH_VERSION = "v0.1.0_hf"
+BIGCODEBENCH_UPDATE = "bigcode/bcb_update"
+BIGCODEBENCH_NEW_VERSION = "v0.1.1"  # TODO(hvaara): [DO NOT MERGE] Figure out which version we're targeting
```

Comment:
Which version on hf.co are we targeting?

Reply:
Based on `fix_v0110.py`, the hf dataset will be `v0.1.2`.

Reply:
Done.